WordPress/twentytwentyfive
# Add Pattern: About Details Descriptions With Bullet

Fixes: https://github.com/WordPress/twentytwentyfive/issues/66


## Screenshot: 

![about-details-descriptions-with-bullets](https://github.com/user-attachments/assets/42ce47c1-b11a-48a8-b5af-4f794cd3a5b2)

## Testing Instructions

- Create a page.
- Add the pattern.
- Confirm that it's looking like the designs (spacing/colors/text etc).
